doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -99,17 +95,21 @@ The following client-side profile is based on the rules set in the schema.
 }
 ```
 
-The schema structure from our example allows filtering the endpoints by the operation system of device (for example, to show only Android devices), os_version (for example, to push some notifications only for the specified os version).
-You can create complex filtering conditions by combining as many filtering conditions as needed.
+The schema structure from this example allows filtering the endpoints by the operation system of device (for example, to show only Android devices), OS version (for example, to push some notifications only for the specified os version).
+You can create complex filters by combining as many filtering conditions as needed.
 
-### Setting client-side EP profile schema from Administration UI
+### Setting client-side EP profile schema
 
-To view the list of client-side endpoint profile schemas created by a [tenant developer]({{root_url}}Glossary/#tenant-developer), open the the **Client-side EP profile** page under the **Schemas** section of the application.
+To view the list of CT schemas used for the client-side endpoint profile, log in to [Administration UI]({{root_url}}Glossary/#administration-ui) and click **Tenant CTL**.
 
-<img src="admin-ui/Client-side endpoint profile schema.png">
+To export a CT schema:
```

Comment:
I don't think exporting a CT schema even belongs here. Move it to CTL page and reference when necessary.
I think we had this in the documentation guide: avoid duplications. This page is **not about** CTL operation.